ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/eea/EeaSendRawTransaction.java

```diff
@@ -171,15 +152,4 @@ JsonRpcErrorResponse getJsonRpcErrorResponse(
     }
```

Comment:
The method `getJsonRpcErrorResponse` can be private :)

Reply:
it is used in both classes!